### PR TITLE
add folders and capsule65

### PR DESCRIPTION
**Pull request by EasonQian1**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/eason/capsule65/config.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+#define MATRIX_ROW_PINS { F4, D1, B7, D0, F5 }
+#define MATRIX_COL_PINS { D5, D2, D3, D4, D6, D7, B4, B5, B6, C6, C7, E2, B3, B1, B0, B2 }
+
+#define DIODE_DIRECTION COL2ROW
+
+#define LED_CAPS_LOCK_PIN F6
+
+#define RGB_DI_PIN F7
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/eason/capsule65/config.h:50]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/eason/capsule65/rules.mk [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/eason/capsule65/rules.mk:18]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by EasonQian1** [comment]
The change has been completed. Please review it. Thank you

**Review by drashna — APPROVED** [review]

File: keyboards/eason/capsule65/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+  [3] = LAYOUT(
+
+  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+  ),
+
```
Comment (fauxpark) [inline keyboards/eason/capsule65/keymaps/default/keymap.c:56]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by EasonQian1** [comment]
Two checks have passed, and merge is requested.

File: keyboards/eason/capsule65/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# capsule65
+
+A customizable 65% keyboard.
+
+* Keyboard Maintainer: [eason](https://github.com/EasonQian1)
+* Hardware Supported: capsule65
+* Hardware Availability: [eason](https://github.com/EasonQian1)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/eason/capsule65/readme.md:9]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/eason/capsule65/readme.md (hunk 4 above)
Comment (EasonQian1) [reply keyboards/eason/capsule65/readme.md]: It has been changed. Thank you very much

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: 键盘/陈奕迅/胶囊65/readme.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# capsule65
```
Comment (zvecr) [inline 键盘/陈奕迅/胶囊65/readme.md:1]: This file has an invalid path, and should be `keyboards/eason/capsule65/readme.md`.

File: keyboards/eason/capsule65/capsule65.h [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT( \
```
Comment (zvecr) [inline keyboards/eason/capsule65/capsule65.h:23]: > A customizable 65% keyboard.

Your layout macro looks a tad questionable, given you are then having to pad with `KC_TRANS`.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/eason/capsule65/capsule65.h (hunk 6 above)
Comment (EasonQian1) [reply keyboards/eason/capsule65/capsule65.h:23]: hello,Can you help me correct it? I don't quite understand.Thank you.

**Review by zvecr — CHANGES_REQUESTED** [review]
```
$ make eason/capsule65:default
QMK Firmware 0.16.5
☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
Making eason/capsule65 with keymap default

☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
avr-gcc (GCC) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
Compiling: keyboards/eason/capsule65/capsule65.c                                                   Compiling: keyboards/eason/capsule65/keymaps/default/keymap.c                                      Compiling: quantum/quantum.c                                                                       Compiling: quantum/send_string.c                                                                   Compiling: quantum/led.c                                                                           Compiling: quantum/action.c                                                                        Compiling: quantum/action_layer.c                                                                  Compiling: quantum/bitwise.c                                                                       keyboards/eason/capsule65/keymaps/default/keymap.c:28:3: error: macro "LAYOUT" passed 80 arguments, but takes just 69
   28 |   ),
      |   ^
In file included from keyboards/eason/capsule65/keymaps/default/keymap.c:17:
keyboards/eason/capsule65/capsule65.h:36: note: macro "LAYOUT" defined here
   36 | #define LAYOUT( \
      | 
keyboards/eason/capsule65/keymaps/default/keymap.c:21:9: error: 'LAYOUT' undeclared here (not in a function)
   21 |   [0] = LAYOUT(
      |         ^~~~~~
keyboards/eason/capsule65/keymaps/default/keymap.c:37:3: error: macro "LAYOUT" passed 80 arguments, but takes just 69
   37 |   ),
      |   ^
In file included from keyboards/eason/capsule65/keymaps/default/keymap.c:17:
keyboards/eason/capsule65/capsule65.h:36: note: macro "LAYOUT" defined here
   36 | #define LAYOUT( \
      | 
 [ERRORS]
 [OK]
 [OK]
 | 
 | 
 | 
make[1]: *** [builddefs/common_rules.mk:452: .build/obj_eason_capsule65_default/keyboards/eason/capsule65/keymaps/default/keymap.o] Error 1
make[1]: *** Waiting for unfinished jobs....
 [OK]
Compiling: quantum/action_tapping.c                                                                Compiling: quantum/action_util.c                                                                    [OK]
 [OK]
 [OK]
 [OK]
 [OK]
 [OK]
Make finished with errors
make: *** [Makefile:413: eason/capsule65:default] Error 1
```

Also line endings of `keyboards/eason/capsule65/config.h` and `keyboards/eason/capsule65/info.json` are bad.

**Comment by EasonQian1** [comment]
> ```
> $ make eason/capsule65:default
> QMK Firmware 0.16.5
> ☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
> Making eason/capsule65 with keymap default
> 
> ☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
> avr-gcc (GCC) 11.2.0
> Copyright (C) 2021 Free Software Foundation, Inc.
> This is free software; see the source for copying conditions.  There is NO
> warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
> 
> ☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
> ☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
> ☒ eason/capsule65: Number of keys for LAYOUT does not match! info.json specifies 80 keys, C macro specifies 69
> Compiling: keyboards/eason/capsule65/capsule65.c                                                   Compiling: keyboards/eason/capsule65/keymaps/default/keymap.c                                      Compiling: quantum/quantum.c                                                                       Compiling: quantum/send_string.c                                                                   Compiling: quantum/led.c                                                                           Compiling: quantum/action.c                                                                        Compiling: quantum/action_layer.c                                                                  Compiling: quantum/bitwise.c                                                                       keyboards/eason/capsule65/keymaps/default/keymap.c:28:3: error: macro "LAYOUT" passed 80 arguments, but takes just 69
>    28 |   ),
>       |   ^
> In file included from keyboards/eason/capsule65/keymaps/default/keymap.c:17:
> keyboards/eason/capsule65/capsule65.h:36: note: macro "LAYOUT" defined here
>    36 | #define LAYOUT( \
>       | 
> keyboards/eason/capsule65/keymaps/default/keymap.c:21:9: error: 'LAYOUT' undeclared here (not in a function)
>    21 |   [0] = LAYOUT(
>       |         ^~~~~~
> keyboards/eason/capsule65/keymaps/default/keymap.c:37:3: error: macro "LAYOUT" passed 80 arguments, but takes just 69
>    37 |   ),
>       |   ^
> In file included from keyboards/eason/capsule65/keymaps/default/keymap.c:17:
> keyboards/eason/capsule65/capsule65.h:36: note: macro "LAYOUT" defined here
>    36 | #define LAYOUT( \
>       | 
>  [ERRORS]
>  [OK]
>  [OK]
>  | 
>  | 
>  | 
> make[1]: *** [builddefs/common_rules.mk:452: .build/obj_eason_capsule65_default/keyboards/eason/capsule65/keymaps/default/keymap.o] Error 1
> make[1]: *** Waiting for unfinished jobs....
>  [OK]
> Compiling: quantum/action_tapping.c                                                                Compiling: quantum/action_util.c                                                                    [OK]
>  [OK]
>  [OK]
>  [OK]
>  [OK]
>  [OK]
> Make finished with errors
> make: *** [Makefile:413: eason/capsule65:default] Error 1
> ```
> 
> `keyboards/eason/capsule65/config.h`和的行尾`keyboards/eason/capsule65/info.json`也不好。

The change has been completed. Please review it. Thank you

**Pull request closed without merging**